ripple/src/main/webapp/script.js

```diff
@@ -129,7 +129,7 @@ function viewAllPostComments() {
 /* Backend for search functionality */
 
 var searchString = document.getElementById("search-bar");
-document.addEventListener('keyup', searchInput);
+searchString.addEventListener('keyup', searchInput);
```

Comment:
Does this work for you? When I changed it like this, none of the search bar entries registered with the 'keyup' event. If it doesn't, I found a way to resolve the issue and can append it to an upcoming PR

Reply:
Nope, this returns null. That would be good